doc/refactorings.md

```diff
@@ -892,6 +893,21 @@ class Hello
 }
 ```
 
+Import Missing Classes
+----------------------
+
+Import all missing classes in the current file.
+
+- **Command**: **RPC Only**
+- **VIM plugin**: _Class context menu > Import Missing
```

Comment:
This underscore makes things italics
```suggestion
- **VIM plugin**: Class context menu > Import Missing
```

Reply:
It should be italics, it's missing the last underscore..